hw/ip/otp_ctrl/data/otp_ctrl_img_dev.hjson

```diff
@@ -20,6 +20,13 @@
                     name:  "CREATOR_SW_CFG_DIGEST",
                     value: "0x0",
                 },
+                {
+                    name: "CREATOR_SW_CFG_USE_SW_RSA_VERIFY",
+                    // Use software mod_exp implementation for signature
+                    // verification. See the definition of `hardened_bool_t` in
+                    // sw/device/lib/base/hardened.h.
+                    value: "0x739",
+                },
```

Comment:
We need to start collecting requirements for an OTP generator tool to be part of `opentitanttool`.

The use of hex constants like this is something we want to avoid long-term: I'd much rather have a human read-able OTP map that said something like:

```
CREATOR_SW_CFG_USE_SW_RSA_VERIFY: true
```
(or `kHardenedTrue`).

_and_ the parser for such a file would have expectations about legal values (ie: it would not permit you to place a value of `0x12345` into this field).

This is fine _for now_, but we need to build some tooling to avoid having to maintain a file full of opaque hex constants.